src/util/get-all-keys.js

```diff
@@ -0,0 +1,10 @@
+
+export default function keys(obj, enumerable = false) {
```

Comment:
I know it may exist in the code elsewhere, but I've been a fan of not falling into the [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap) and using an options object instead.

You could define this as:

```js
keys(obj, { enumerable = false });
```